pkg/server/router.go

```diff
@@ -34,6 +35,9 @@ func router(serverConfig *config.Control, tunnel http.Handler, cacertsGetter CAC
 	router.Path("/openapi/v2").Handler(serveOpenapi())
 	router.Path("/ping").Handler(ping())
 
+	staticDir := filepath.Join(serverConfig.DataDir, "static")
```

Comment:
cool.  Can you just move this out to a different method so that we keep the pretty format.  basically

```go
router.PathPrefix("/static").Handler(static(staticDir))
```